contract/src/Custody.sol

```diff
@@ -5,14 +5,26 @@ import {IChannel} from "./interfaces/IChannel.sol";
 import {IDeposit} from "./interfaces/IDeposit.sol";
 import {IAdjudicator} from "./interfaces/IAdjudicator.sol";
 import {IComparable} from "./interfaces/IComparable.sol";
-import {Channel, State, Allocation, Status, Signature, Amount, CHANOPEN, CHANCLOSE} from "./interfaces/Types.sol";
+import {
+    Channel,
+    State,
+    Allocation,
+    Status,
+    Signature,
+    Amount,
+    CHANOPEN,
+    CHANCLOSE,
+    CREATOR,
+    BROKER
```

Comment:
It seems that you have not pushed changes to `Types.sol` file. Nevertheless, I think CREATOR and BROKER should be defined in Custody

Reply:
Also, I think we better name them `USER` and `BROKER` or `CREATOR` and `RESPONDER` (or else). I don't want to mix up the action ("create") and a role ("broker").